### PR TITLE
replace "dependencies are completely removed"

### PR DESCRIPTION
**Pull request by franzholz**
How can dependencies be removed?

### CODE REVIEW

File: Documentation/Installation/Install.rst [hunk 1]
```diff
@@ -13,10 +13,11 @@ Welcome to the TYPO3 installation guide. This guide covers each of the steps req
 Installation And Deployment
 ---------------------------
 
-Composer should only ever be used to install TYPO3 on a local or development environment. By not using Composer in a live environment, the need to install
-dependencies are completely removed - reducing potential vectors for attack.
+Composer should only ever be used to install TYPO3 on a local or development environment.
+By **not using Composer** in a live environment there will be no need for installing
+dependencies - reducing potential vectors for attack.
```
Comment (gilbertsoft) [inline Documentation/Installation/Install.rst:18]: The whole part is still very confusing. Not sure how Composer on the server should increase attack vectors at all. But it's a good practise to use Composer locally or on the deployment environment only, that's correct so far.

File: Documentation/Installation/Install.rst [hunk 2]
```diff
@@ -165,7 +166,7 @@ TYPO3 can either connect to an existing empty database or create a new one.
 The list of databases available is dependent on which database drivers are installed on the host.
 
 For example, if an instance of TYPO3 is intended to be used with a MySQL database then the PHP extension 'pdo_mysql' is required.
-Once it's installed, 'MySQL Database' will be an available option.(Review)
+Once it's installed, 'MySQL Database' will be an available option. (Review)
```
Comment (gilbertsoft) [inline Documentation/Installation/Install.rst:169]: The review at the end should be removed once this behavior is confirmed!

File: Documentation/Installation/Install.rst (hunk 1 above)
Comment (susannemoog) [reply Documentation/Installation/Install.rst:18]: Probably not composer "per se" - but composer on the server would tempt people to install dev dependencies or do requires and testing directly on the live system.

Comment (franzholz) [reply Documentation/Installation/Install.rst:18]: Composer fetches software from the internet. When a software update is made, then the dependent extensions will be fetched. The server with the software sources could be compromised. 
Or the composer file could be manipulated without any notice. 

If you copy everything from your local computer, then this should be safer. Then you can be sure, that both have the same software. Your local computer is not exposed to the internet attackers.

Comment (gilbertsoft) [reply Documentation/Installation/Install.rst:18]: > Probably not composer "per se" - but composer on the server would tempt people to install dev dependencies or do requires and testing directly on the live system.

Agree, but that's not was is written here.

Comment (linawolf) [reply Documentation/Installation/Install.rst]: I tried to make the suggestions more specific here. please see if we can get it merged like this now.

**Review by gilbertsoft — APPROVED** [review]

**Review by linawolf — APPROVED** [review]

**Pull request merged**